appserver/payara-appserver-modules/microprofile/openapi/src/main/java/fish/payara/microprofile/openapi/impl/rest/app/OpenApiApplication.java

```diff
@@ -40,6 +40,7 @@
 package fish.payara.microprofile.openapi.impl.rest.app;
 
 import static fish.payara.microprofile.openapi.impl.rest.app.OpenApiApplication.OPEN_API_APPLICATION_PATH;
+import fish.payara.microprofile.openapi.impl.rest.app.provider.CorsHeadersFilter;
```

Comment:
Tiny issue, but could you move this import to be with the other similar imports?

Reply:
Moved the import next to the other similar imports.